linodecli/baked/operation.py

```diff
@@ -338,6 +363,14 @@ def parse_args(
             for arg in self.args:
                 if arg.read_only:
                     continue
+
+                arg_type_handler = TYPES[
+                    arg.item_type if arg.datatype == "array" else arg.datatype
+                ]
```

Comment:
I think the `array` is already handled below by the if...else... statement?

Reply:
So this check is necessary because arrays need to use `arg.item_type` for argparse rather than `arg.datatype`. I moved this out of the `parser.add_argument` call so it could be filtered through the `wrap_parse_nullable_value` function.

Reply:
The array seems to be handled here?
https://github.com/linode/linode-cli/pull/421/files#diff-f69e89596ddac39d63389afca33deb5535e8678cd509684f9d89734007c2f096R375-R382